crates/sui-swarm/src/memory/container-sim.rs

```diff
@@ -56,12 +56,12 @@ impl Container {
             .name(&format!("{:?}", config.protocol_public_key().concise()))
             .init(move || {
                 info!("Node restarted");
-                let config = config.clone();
+                let config = (*config.clone()).clone();
```

Comment:
Is the 2nd `clone()` necessary?

Reply:
I believe so because it is in an Arc

```
error[E0507]: cannot move out of an `Arc`
  --> crates/sui-swarm/src/memory/./container-sim.rs:59:30
   |
59 |                 let config = *config.clone();
   |                              ^^^^^^^^^^^^^^^ move occurs because value has type `NodeConfig`, which does not implement the `Copy` trait
```